subsys/fw_info/Kconfig

```diff
@@ -22,6 +22,7 @@ config FW_INFO_OFFSET
 
 config FW_INFO_FIRMWARE_VERSION
 	int "Version number of the firmware."
+	range 0 32766 if SB_MONOTONIC_COUNTER
```

Comment:
A comment here explaning why would be great.

Reply:
Added help text.